src/frontend/src/handler/create_sql_function.rs

```diff
@@ -204,7 +208,7 @@ pub async fn handle_create_sql_function(
                     }
                 }
                 Err(e) => return Err(ErrorCode::InvalidInputSyntax(format!(
-                    "failed to conduct semantic check, please see if you are calling non-existence functions: {}",
+                    "failed to conduct semantic check, please see if you are calling non-existence functions or parameters\ndetailed error message: {}",
```

Comment:
This is just a current work around, I prefer a better hint display in the future, related: #14853.